hazelcast-integration/spring-configuration/src/main/java/com/hazelcast/springconfiguration/HazelcastDataTypes.java

```diff
@@ -181,17 +179,6 @@ private static void executeSemaphore() {
         System.out.println("Available semaphore permits: " + semaphore.availablePermits() + "\n");
     }
 
-    private static void executeLock() {
-        System.out.println("### Lock Execution Started... ###");
-        FencedLock lock = context.getBean("lock", FencedLock.class);
-        lock.lock();
-        System.out.println("lock() call...");
-        System.out.println("is locked by current thread: " + lock.isLockedByCurrentThread());
-        lock.unlock();
-        System.out.println("unlock() call...");
-        System.out.println("is locked: " + lock.isLocked());
-    }
-
```

Comment:
Is there a reason you removed `FencedLock` while keeping the others?

Reply:
My understanding is that this part has changed since the introduction of CP subsystem.